pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -31,7 +32,7 @@ type Client struct {
 }
 
 // NewClient creates new KUDO Client
-func NewClient(kubeConfigPath string, requestTimeout int64) (*Client, error) {
+func NewClient(kubeConfigPath string, requestTimeout int64, validateInstall bool) (*Client, error) {
```

Comment:
I wouldn't mind adding a test for this method actually verifying the behavior with validate=true 👼

Reply:
I've added a unit test, but I have a hard time coming up with a test in the current structure. I'll start a refactoring PR next, and going to tackle the whole testing thing there.